### ISSUE TITLE
objective.direction not read from SBML (minimize objectives are maximized)

### ISSUE THREAD

**Issue by matthiaskoenig** [opencobra/cobrapy#480]
I am not sure, but it looks like the objective.direction is just set to `maximize` independent of the direction of the active objective in the SBML file.

The information about objective direction is stored in the attribute fbc:type
```
<fbc:objective fbc:id="obj1" fbc:type="maximize">
...
<fbc:objective fbc:id="obj2" fbc:type="minimize">
```
In the current form all minimize objectives are maximized in cobrapy.

In `cobra.io.sbml3.py` the objective direction must be read from the SBML active objective
and used in creating the objective.
I.e.
```
 set_objective(model, coefficients, direction)
```

Current code
```
    # objective coefficients are handled after all reactions are added
    obj_list = xml_model.find(ns("fbc:listOfObjectives"))
    if obj_list is None:
        warn("listOfObjectives element not found")
        return model
    target_objective = get_attrib(obj_list, "fbc:activeObjective")
    obj_query = OBJECTIVES_XPATH % target_objective
    coefficients = {}
    for sbml_objective in obj_list.findall(obj_query):
        rxn_id = clip(get_attrib(sbml_objective, "fbc:reaction"), "R_")
        try:
            objective_reaction = model.reactions.get_by_id(rxn_id)
        except KeyError:
            raise CobraSBMLError("Objective reaction '%s' not found" % rxn_id)
        try:
            coefficients[objective_reaction] = get_attrib(
                sbml_objective, "fbc:coefficient", type=number)
        except ValueError as e:
            warn(str(e))
    set_objective(model, coefficients)
    return model
```

**Comment by hredestig** [comment]
What do you reckon should be the behavior if the list of objectives is longer than 1?

**Comment by matthiaskoenig** [comment]
I would only read the active one and give warnings that not all read.
Perhaps add a helper function which allows to read all flux objectives in
combination with a class to store it and a helper to set individual flux
objectives on a model. This would allow to switch between objectives easily.

**Comment by hredestig** [comment]
closed with #489
